src/components/captcha.js

```diff
@@ -1,17 +1,44 @@
-import React from 'react';
-import ReCAPTCHA from 'react-google-recaptcha';
-
-const Captcha = (props) => (
-  <div>
-    <ReCAPTCHA
-      sitekey={process.env.RECAPTCHA_SITE_KEY}
-      onChange={response => props.input.onChange(response)}
-    />
-    </div>
-);
-
-Captcha.propTypes = {
-  input: React.PropTypes.object.isRequired
-};
-
-export default Captcha;
+import React, {Component} from 'react';
+import Recaptcha from 'react-recaptcha';
+import * as actions from '../actions/index';
+import {connect} from 'react-redux';
+
+
+const callback = function (){
+  console.log ('re-captcha loaded')
+}
+
+class Captcha extends Component {
+  constructor(props){
+    super(props)
+    this.verifyCallback = this.verifyCallback.bind(this)
+  }
+
+  verifyCallback (response){
+    if(response){
+      this.props.dispatch(
+        actions.recaptchaVerification('not a robot')
+      )
+    }
+  }
+
+  render() {
+    return (
+      <div className="g-recaptcha">
+        <Recaptcha
+          sitekey="6Ld0fiIUAAAAAG7rGM4RCiYBkKbrJAqmgUVbqe_7"
```

Comment:
I don't think it should either. I tried putting the key on as an environment variable from an env.sh file, but it didn't work. I think it is because I am using an npm library, so I have to figure out how to use an environment variable in this particular field (sitekey field).